### ISSUE TITLE
which one is recommeded to remove?

### ISSUE THREAD

**Issue by Najierida** [jumphone/SPRINT#13]
In your  paper , there a sentence "suggesting that many one-read-count SNVs may be system errors", 
So in the SPRINT output files "SPRINT_identified_all.res", which SNVs should be remove? 
the SNVs with 1 supporting read, or the SNVs which AD:DP = 1:1 ?

Thank you.

**Comment by jumphone** [comment]
Hi,

In the paper, the whole sentance is: 

“In comparison, the A-to-G rate of the RESs called by filtering all SNVs is only 76.5%, suggesting that many one-read-count SNVs may be system errors.”

Before using clustering (SNP-free) strategy, we have tested the idea of getting RESs by filtering out SNPs from the SNVs called by SPRINT. 

This sentance describes that SNVS filtering (SNP removing) strategy cannot be used to detect RESs from one-read-count SNVs (low A-to-G rate), because there are a lot of system errors.

In contrast, clustering strategy (SNP-free) can achieve a high A-to-G rate when handling one-read-count SNVs (please see Supplementary Figure 4).

In conclusion, those RESs with 1:1 (AD:DP) are also in an acceptable quality (because of the high A-to-G rate). However, people may want to analyze RESs with high depth, and you can set cutoffs for AD, DP, and AD/DP, separately.

**Comment by Najierida** [comment]
Thank you for your detailed reply.
